src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageChange = (page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={(event) => {
```

Comment:
It is usually not a good practice to use `href="#prev"` as it can cause the page to jump to the top when clicked. Consider using `href="javascript:void(0);"` or better yet, use a button instead of a link for better semantics since this is not really a navigation link.